src/db/data_insertion.py

```diff
@@ -1,9 +1,10 @@
 from src.db.file_transfer import ftp_transfer
 from src.db.db_connection import sql_insert, IMAGE, BBOX
+from decimal import Decimal
 import datetime
 
 
-def data_insert(image_name: str, image_date: datetime, image_time: datetime, init_vecs: list[str], bboxes: list[int], input_dir: str):
+def data_insert(image_name: str, image_date: datetime, image_time: datetime, init_vecs, bboxes, input_dir: str):

```

Comment:
why are we removing type hints here? If the type changed so should the type hints.

Reply:
The syntax of these type hints was incorrect. Will change to `list` for both